delfin/api/v1/volume_groups.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from delfin import db
+from delfin.api import api_utils
+from delfin.api.common import wsgi
+from delfin.api.views import volume_groups as volume_group_view
+
+
+class VolumeGroupController(wsgi.Controller):
+
+    def __init__(self):
+        super(VolumeGroupController, self).__init__()
+        self.search_options = ['name', 'id', 'storage_id',
```

Comment:
filter for volumes?

Reply:
Need to explore this option, created an issue to track this (https://github.com/sodafoundation/delfin/issues/624)